src/main/java/com/sourcery/gymapp/backend/workout/dto/ResponseRoutineExerciseDto.java

```diff
@@ -0,0 +1,16 @@
+package com.sourcery.gymapp.backend.workout.dto;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+public record ResponseRoutineExerciseDto(
+        UUID routineExerciseId,
+        ExerciseSimpleDto exercise,
```

Comment:
Same as Dto explenation

I guess its still better to return all information of exercises, maybe we would like to have option to read more information about our exercise whenever we get lists of our routine exercises